ppr-ui/src/composables/fees/FeeSummary.vue

```diff
@@ -145,17 +149,25 @@ export default defineComponent({
         if (localState.feeType === FeeSummaryTypes.RENEW) {
           feeSummary.processingFee = 5
         }
+        if (props.setFeeOverride) {
+          feeSummary.feeAmount = props.setFeeOverride.feeAmount
+        }
+        if (props.setFeeOverride && feeSummary.serviceFee !== 0) {
```

Comment:
Is there a circumstance where we want to override the service fee to 0?

Reply:
Not yet! For now we always override it to what their custom service fee is except for cases where there is no service fee